hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerScrubber.java

```diff
@@ -130,12 +128,13 @@ private void throttleScrubber(TimeStamp startTime) {
 
   private void scrub() {
 
-    Iterator<Container> containerIt = containerSet.getContainerIterator();
+    Iterator<Container> containerIt = controller.getContainerSetIterator();
     long count = 0;
 
     while (containerIt.hasNext()) {
       TimeStamp startTime = new TimeStamp(System.currentTimeMillis());
       Container container = containerIt.next();
+      Handler containerHandler = controller.getHandler(container);
```

Comment:
containerHandler is not used here. Is it for upcoming patches?

Reply:
Yes @hanishakoneru it is for later use in the Directory scanner part.